benchmarking/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "kakarot-rpc-benchmarking",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "type": "commonjs",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "format": "npx prettier --write .",
+    "metrics": "node ./scripts/metrics.js",
+    "run-madara": "RUST_LOG=error ../lib/madara/target/release/madara --dev",
```

Comment:
when is madara built actually?

Reply:
In the CI, `benchmark.yml`.

<img width="381" alt="image" src="https://github.com/kkrt-labs/kakarot-rpc/assets/41180869/cc41c111-dc05-45f2-9fb3-e825849d2e03">